### ISSUE TITLE
faker.geo_coordinate() should accept Decimals as arguments

### ISSUE THREAD

**Issue by mrabbitt** [joke2k/faker#253]
`faker.geo_coordinate()` returns a `Decimal`, but it doesn't work if you attempt to use `Decimal`'s as arguments:

```
>>> from faker import Factory
>>> faker = Factory.create()
>>> from decimal import Decimal
>>> faker.geo_coordinate(50.5, 0.1)
Decimal('50.5866123539')
>>> faker.geo_coordinate(Decimal(50.5), Decimal(0.1))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/mrabbitt/.virtualenvs/data_migration/lib/python2.7/site-packages/faker/providers/address/__init__.py", line 91, in geo_coordinate
    geo = random.uniform(center - radius, center + radius)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/random.py", line 360, in uniform
    return a + (b-a) * self.random()
TypeError: unsupported operand type(s) for *: 'Decimal' and 'float'
```

**Comment by wybiral** [comment]
Does #280 close this issue?

**Comment by fcurella** [comment]
Yes, this should be fixed in d08fd3d12d9bd819555ee36d21e261208e6203fe. Thanks @wybiral :)
